homeassistant/components/subaru/sensor.py

```diff
@@ -14,128 +20,133 @@
-        SENSOR_UNITS: ELECTRIC_POTENTIAL_VOLT,
-    },
+    SensorEntityDescription(
+        key=sc.AVG_FUEL_CONSUMPTION,
+        icon="mdi:leaf",
+        name="Avg Fuel Consumption",
+        native_unit_of_measurement=FUEL_CONSUMPTION_LITERS_PER_HUNDRED_KILOMETERS,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=sc.DIST_TO_EMPTY,
+        icon="mdi:gas-station",
+        name="Range",
+        native_unit_of_measurement=LENGTH_KILOMETERS,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=sc.TIRE_PRESSURE_FL,
+        device_class=SensorDeviceClass.PRESSURE,
+        name="Tire Pressure FL",
+        native_unit_of_measurement=PRESSURE_HPA,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=sc.TIRE_PRESSURE_FR,
+        device_class=SensorDeviceClass.PRESSURE,
+        name="Tire Pressure FR",
+        native_unit_of_measurement=PRESSURE_HPA,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=sc.TIRE_PRESSURE_RL,
+        device_class=SensorDeviceClass.PRESSURE,
+        name="Tire Pressure RL",
+        native_unit_of_measurement=PRESSURE_HPA,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=sc.TIRE_PRESSURE_RR,
+        device_class=SensorDeviceClass.PRESSURE,
+        name="Tire Pressure RR",
+        native_unit_of_measurement=PRESSURE_HPA,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=sc.EXTERNAL_TEMP,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        name="External Temp",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key=sc.BATTERY_VOLTAGE,
+        device_class=SensorDeviceClass.VOLTAGE,
+        name="12V Battery Voltage",
+        native_unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
 ]
 
-# Sensor data available to "Subaru Safety Plus" subscribers with PHEV vehicles
+# Sensors available to "Subaru Safety Plus" subscribers with PHEV vehicles
 EV_SENSORS = [
-    {
-        SENSOR_TYPE: "EV Range",
-        SENSOR_CLASS: None,
-        SENSOR_FIELD: sc.EV_DISTANCE_TO_EMPTY,
-        SENSOR_UNITS: LENGTH_MILES,
-    },
-    {
-        SENSOR_TYPE: "EV Battery Level",
-        SENSOR_CLASS: SensorDeviceClass.BATTERY,
-        SENSOR_FIELD: sc.EV_STATE_OF_CHARGE_PERCENT,
-        SENSOR_UNITS: PERCENTAGE,
-    },
-    {
-        SENSOR_TYPE: "EV Time to Full Charge",
-        SENSOR_CLASS: SensorDeviceClass.TIMESTAMP,
-        SENSOR_FIELD: sc.EV_TIME_TO_FULLY_CHARGED,
-        SENSOR_UNITS: TIME_MINUTES,
-    },
+    SensorEntityDescription(
+        key=sc.EV_DISTANCE_TO_EMPTY,
+        icon="mdi:ev-station",
+        name="EV Range",
```

Comment:
In a follow up please update the description names to follow our guidelines for lowercase and uppercase letters.

https://developers.home-assistant.io/docs/core/entity#entity-naming

We should also migrate the entity unique_id to not use the description name but the description key as suffix.